crates/miden-lib/src/account/faucets/basic_fungible.rs

```diff
@@ -42,7 +42,7 @@ procedure_digest!(
 );
 
 static METADATA_SLOT_NAME: LazyLock<StorageSlotName> = LazyLock::new(|| {
-    StorageSlotName::new("miden::basic_fungible_faucet::metadata")
+    StorageSlotName::new("miden::standards::faucets::metadata")
```

Comment:
Since this is now used for both basic and network faucets, how about moving this definition to `crates/miden-lib/src/account/faucets/mod.rs`?